build/local_kubernetes.sh

```diff
@@ -59,6 +60,29 @@ start_localkube() {
     kind get kubeconfig --name="kanister" > "${HOME}/.kube/config"
     wait_for_nodes
     wait_for_pods
+
+    SNAPSHOTTER_VERSION=v5.0.0
+
+    # Install VolumeSnapshot CRDs
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml
```

Comment:
let's see if we can use one kubectl statement to create all these resources. Can a syntax like this be used
https://github.com/digitalocean/csi-digitalocean#3-deploy-the-csi-plugin-and-sidecars

Reply:
Yeah. We can merge the kubectl commands in the way done on csi-digitalocean.